tools/quay-registry.sh

```diff
@@ -9,18 +9,22 @@ CLUSTER="${CLUSTER:-OPENSHIFT}"
 MARKETPLACE_NAMESPACE="${MARKETPLACE_NAMESPACE:-openshift-marketplace}"
 PACKAGE="${PACKAGE:-hco-operatorhub}"
 APP_REGISTRY_NAMESPACE="${APP_REGISTRY_NAMESPACE:-kubevirt-hyperconverged}"
+TARGET_NAMESPACE="${TARGET_NAMESPACE:-kubevirt-hyperconverged}"
+
+# Latest version from: https://quay.io/application/kubevirt-hyperconverged/hco-operatorhub
```

Comment:
I don't see anything at this URL.

Reply:
It's the old url.  Should be https://quay.io/application/kubevirt-hyperconverged/kubevirt-hyperconverged